### ISSUE TITLE
Windows store protocol

### ISSUE THREAD

**Issue by necrose99** [chocolatey/choco#1823]
choco install-wstore -y [yubikey-hello](https://www.microsoft.com/en-us/p/yubikey-for-windows-hello/9nblggh511m5)   
windows iot remote client , [windows 10 Configuration designer ](https://www.microsoft.com/en-us/p/windows-configuration-designer/9nblggh4tx22)
ASUS ROG or other MFR laptop apps etc.. 

choco wstore-ban candycrush*  crappy/trash games... or other annoying pkgs... 

deploy via boxstater etc..   and or regualr chco... as req.. 

deploy windows store apps via chocolatey / or virtual nuget pkgs with less hells... 

A choco : windows store extension , or wrapper.  to treat windows store items as choco packages or virtual pkgs.. wich choco can thus automate.. find and deploy..

**Comment by ferventcoder** [comment]
@necrose99 we've been talking about a want for something like this for awhile. The problem exists in that there is no way to call for the windows store item without knowing the internal url to download it and side load it from. So this is blocked by Windows Store needing to implement some features.

**Comment by pauby** [comment]
#3360 is more fleshed out so I'm going to close this one as a duplicate of that, despite this one being here first.
